code/game/machinery/computer/arcade.dm

```diff
@@ -341,6 +341,15 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 
 // *** THE ORION TRAIL ** //
 
+/obj/item/gamer_pamphlet
+	name = "pamphlet - \'Violent Video Games and You\'"
+	desc = "A pamphlet encouraging the reader to maintain a balanced lifestyle and take care of their mental health, while still enjoying video games in a healthy way. You probably don't need this..."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "pamphlet"
+	item_state = "paper"
+	w_class = WEIGHT_CLASS_TINY
```

Comment:
Could we make this an actual paper subtype?

Reply:
You can't actually read it because I don't want to write an entire pamphlet of fluff for this, so I feel like being paper would be a waste

Reply:
It's not paper because you can't actually read it, because I don't want to write an entire pamphlet for a throwaway joke. Making it a child of paper and removing the ability to read it seemed weirder than just making it a useless item